README.md

```
@@ -1,7 +1,12 @@
-# wp-omnisend-lifterlms
+# Omnisend for LifterLMS Add-On
 
-Plugin for _LifterLMS_ WordPress plugin. More information can be found [here](https://wordpress.com/plugins/lifterlms).
+Automatically syncs LifterLMS contacts, new subscribers, their courses, and membership levels to Omnisend.
+
+Requires the [Omnisend](https://wordpress.org/plugins/omnisend/) plugin.
 
+WordPress plugin: https://wordpress.org/plugins/omnisend-for-lifterlms-add-on/
+
+Plugin for _LifterLMS_ WordPress plugin. More information can be found [here](https://wordpress.com/plugins/lifterlms).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)